scripts/build.sh

```diff
@@ -6,8 +6,8 @@ set -e
 
 (
     cd cdk
-    npm ci
-    npm run lint
-    npm test
-    npm run synth
+    bun install --frozen-lockfile
```

Comment:
See https://github.com/oven-sh/bun/issues/5283 and https://github.com/oven-sh/bun/pull/16137